packages/patternfly-4/content/design-guidelines/usage-and-behavior/about-modal/design.md

```diff
@@ -18,7 +18,10 @@ The About Modal stores the version number(s) of the application as well as any a
 7. Close
 8. Modal image: The modal image should be the same as the [background image](/documentation/react/components/backgroundimage) you use for your application.
 
-## Components and demos used
+## Usage
+Use the About Modal to provide information about the product including the product name, logo, version numbers, and any appropriate legal text.
+
+## Related components and demos
```

Comment:
I didn't understand what we had here. Components and Related component? Seems redundant. Maybe just an editing error?

Reply:
@mcarrano - I think I fixed it